metrotranscom/doorway
# feat: add footer to the listings page

## Description

- Add footer back into the listings page, both for mobile and desktop.
- The footer is not displayed for the map only page on mobile.

UXD pointed out the [Airbnb listings page](https://www.airbnb.com/s/Italy/homes?tab_id=home_tab&refinement_paths%5B%5D=%2Fhomes&flexible_trip_lengths%5B%5D=one_week&monthly_start_date=2023-07-01&monthly_length=3&price_filter_input_type=0&price_filter_num_nights=2&channel=EXPLORE&place_id=ChIJA9KNRIL-1BIRb15jJFz1LOI&date_picker_type=calendar&checkin=2023-06-22&checkout=2023-06-24&adults=1&source=structured_search_input_header&search_type=user_map_move&drawer_open=false&query=Italy&ne_lat=43.27235140628151&ne_lng=-1.9638244388750081&sw_lat=33.96516889752949&sw_lng=-10.005816626375008&zoom=7&zoom_level=7&search_by_map=true) as an example of the ideal layout. The difference between that and this PR is that the Airbnb footer is displayed in a continuous scroll from the listings list, whereas here we have to move the mouse slightly for the footer to come into view. This is controlled via the overscroll-behavior property https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior and the default value (auto) is already what we want. I'm not sure we can accomplish that behavior without more substantial changes because their scrollbar, for example, is for the whole page and not for the listings list div.

## How Can This Be Tested/Reviewed?

- Bring up the public site.
- Go to the listings page on desktop. Scroll down to the bottom of the listings list, move your mouse slightly and then keep scrolling. The footer should come into view. You won't see the footer if you keep scrolling after hitting the end of the listings list, you just need to move your mouse slightly for it to work.
- Switch to mobile, reload the page if needed to see the hybrid listings page. Scroll down in the listings list part of it, and once you reach the bottom scroll again and you should see the footer.
- Swipe the listings list up to go to the list only view. Scroll down again and you should see the footer at the bottom.
- Swipe down on the listings list, and then again from the hybrid page to get to the map only view. Try scrolling down - there should be no vertical scroll and no footer.
